### PR TITLE
Switch default reproduction-template to app

### PR DESCRIPTION
**Pull request by samcx**
### Updating reproduction examples

Our current `pnpm create next-app -e reproduction-template`, which is present in our Issues template, will create a Pages app. This PR updates the default (`reproduction-template`) to App Router, and adds a `reproduction-template-pages` example.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: bc5b6aaf1da2a8e8cd390b61976b4bf3897104e9

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: examples/reproduction-template/README.md [hunk 1]
```diff
@@ -33,12 +34,12 @@ To learn more about Next.js, take a look at the following resources:
 - [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 - [How to Contribute to Open Source (Next.js)](https://www.youtube.com/watch?v=cuoNzXFLitc) - a video tutorial by Lee Robinson
 - [Triaging in the Next.js repository](https://github.com/vercel/next.js/blob/canary/contributing.md#triaging) - how we work on issues
-- [CodeSandbox](https://codesandbox.io/s/github/vercel/next.js/tree/canary/examples/reproduction-template) - Edit this repository on CodeSandbox
+- [CodeSandbox](https://codesandbox.io/s/github/vercel/next.js/tree/canary/examples/reproduction-template-app-dir) - Edit this repository on CodeSandbox
```
[the remark this answers is not in the document]
Comment (samcx) [reply examples/reproduction-template/README.md:37]: @styfle Shouldn't it be fine? I updated the directory names.

[the remark this answers is not in the document]
Comment (styfle) [reply examples/reproduction-template/README.md:37]: @samcx Take a look again.

The name of this file is `examples/reproduction-template/README.md`.

Therefore the url to "edit this repository" should be `examples/reproduction-template`

[the remark this answers is not in the document]
Comment (samcx) [reply examples/reproduction-template/README.md:37]: Ahh different area, I forgot to include change the code sandbox ones

**Review by styfle — DISMISSED** [review]

**Review by styfle — APPROVED** [review]

**Pull request merged**